docs-md/concepts/collections/overview.md

```diff
@@ -1,7 +1,35 @@
-Although an individual event represents vitally important information, in practice they are almost more useful when grouped together with related events. ksqlDB supports durable collections of events that can be replicated across multiple servers. This means that, similar to replicated Postgres, that if there is a recoverable failure of some of the servers, your events remain permanently stored. It is able to do this by storing data in Kafka topics.
+---
+layout: page
+title: Collections
+tagline:  Streams and tables in ksqlDB
+description: Learn about durable collections of events in ksqlDB
+keywords: ksqlDB, events, collection, stream, table
+---
 
-Collections are sharded, and can arbitrarily grow in size by adding more Kafka brokers to a ksqlDB cluster. This sharding is more commonly known as “partitioning” in Kafka, and is the term that ksqlDB uses as well. Events with the same key are stored on the same partition.
+Although an individual [event](../events.md) represents vitally important
+information, events are more useful when grouped together with related events.
+ksqlDB supports durable collections of events that are  replicated across
+multiple servers. This means that, similar to replicated
+[Postgres](https://www.postgresql.org/), if there is a recoverable failure
+of some of the servers, your events remain permanently stored. ksqlDB can
+do this by storing data in {{ site.aktm }} topics.
 
-ksqlDB offers multiple abstractions for storing events. Despite the fact that a single event is immutable, collections of events can either model immutability or mutability to represent change over time. In addition, new collections can be derived from existing ones. Derived collections are kept up to date in real-time, which is the heart of stream processing.
+Collections are sharded and can grow to arbitrary sizes by adding more
+{{ site.ak }} brokers to a ksqlDB cluster. This sharding is more commonly
+known as *partitioning* in {{ site.ak }} and is the term that ksqlDB uses
+as well. Events with the same key are stored on the same partition. For more
+information, see
+[Partition Data to Enable Joins](../../developer-guide/partition-data.md).
 
-Collections are represented as a series of rows and columns which have a defined schema. Only data that conforms to the schema can be added to the collection.
\ No newline at end of file
+ksqlDB offers multiple abstractions for storing events. Although a single
+event is immutable, collections of events can model either immutability or
```

Comment:
are you viewing for example updating a table as a remove and add to the collection, while the event itself remains unchanged?

Reply:
I was trying to say that when you work with a table, you have an abstraction that gives you mutability in the sense it supports "updates", where streams only support "appends".